cypress/support/pages/home.pageObject.js

```diff
@@ -1,10 +1,38 @@
 import PageObject from '../PageObject';
 
 class HomePageObject extends PageObject {
-  url = '/#/';
+  url = '#/';
```

Comment:
The URL should start with '/#/' to correctly navigate to the home page, not with '#/'.